apps/fyle/tasks.py

```
@@ -19,7 +20,7 @@
 from apps.fyle.models import Expense, ExpenseFilter, ExpenseGroup, ExpenseGroupSettings
 from apps.tasks.models import TaskLog, Error
 from apps.workspaces.actions import export_to_qbo
-from apps.workspaces.models import FyleCredential, Workspace, WorkspaceGeneralSettings
+from apps.workspaces.models import FyleCredential, LastExportDetail, Workspace, WorkspaceGeneralSettings
```

Ruff (0.8.2)

`apps.workspaces.models.LastExportDetail` imported but unused

Remove unused import: `apps.workspaces.models.LastExportDetail`

(F401)